lib/pharos/cluster_manager.rb

```diff
@@ -13,7 +13,9 @@ class ClusterManager
     def initialize(config, pastel: Pastel.new)
       @config = config
       @pastel = pastel
-      @context = {}
+      @context = {
+        'post_install_messages' => {}
```

Comment:
and this could then be `'post_install_messages' => Hash.new { |hash,key| hash[key] = [] }` so you don't need the `||= []` in the `post_install_message` method